src/renderers/dom/shared/utils/__tests__/setInnerHTML-test.js

```diff
@@ -24,17 +24,35 @@ describe('setInnerHTML', () => {
   });
 
   describe('when the node does not have an innerHTML property', () => {
-    // Disabled. JSDOM doesn't seem to remove nodes when using appendChild to
-    // move existing nodes.
-    xit('sets innerHTML on it', () => {
+    var node;
+    var nodeProxy;
+    beforeEach(() => {
       // Create a mock node that looks like an SVG in IE (without innerHTML)
-      var node = {
-        namespaceURI: Namespaces.svg,
-        appendChild: jasmine.createSpy(),
-      };
+      node = document.createElementNS(Namespaces.svg, 'svg');
 
+      nodeProxy = new Proxy(node, {
+        has: (target, prop) => {
+          return prop === 'innerHTML' ? false : prop in target;
+        },
+        get: (target, prop) => {
+          if (prop === 'appendChild') {
```

Comment:
Why do we need this? If you append a child to another parent, it will be removed from the current parent.

Isn't just 

```js
        get: (target, prop) => {
          return target[prop];
        },
```

enough?

Reply:
In the comment [here](https://github.com/facebook/react/pull/11108/files#diff-58de46abac1f1a2608d82f6fb7b1d2b4L27) it says that JSDOM doesn't remove the node from the current parent when moving nodes using `appendChild`, though I didn't verify this myself.

I could check and see if that's still relevant, if not then I can remove the `get` handler altogether.

Reply:
Please check this, yes. The comments might be very outdated.

Reply:
You were right, this was no longer necessary 🎉 
Removed the `get` handler altogether!